ros_ws/src/capstone_interfaces/srv/PlannerQuery.srv

```diff
@@ -0,0 +1,6 @@
+string question
+---
+string location
```

Comment:
What's a query look like realistically? Just trying to picture the interaction here.